libs/voicefocus/worklet-inline-node.js

```diff
@@ -11,11 +14,13 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, ge
 Object.defineProperty(exports, "__esModule", { value: true });
 const support_js_1 = require("./support.js");
 const types_js_1 = require("./types.js");
+const CPU_WARNING_MAX_INTERVAL_MILLISECONDS = 60 * 1000;
```

Comment:
nit: We usually just use MS instead

Reply:
Is 60s a bit too long?

Reply:
I think 60 is ok, but I can change it to 1 or 10 to make it more apparent to builders in logs that voice focus is not working well due to CPU usage.